testfiles/AirflowNetwork_MultiZone_SmallOffice.idf

```diff
@@ -1995,6 +1995,11 @@
     0.01,                    !- Air Mass Flow Coefficient at Reference Conditions {kg/s}
     0.667;                   !- Air Mass Flow Exponent {dimensionless}
 
+  AirflowNetwork:MultiZone:SpecifiedFlowRate,
+    SF-1,                    !- Name
+    0.06,                    !- Air Flow Value
+    kg/s;                    !- Air Flow Units
+
```

Comment:
It is better to add output variables to demonstrate this new feature. For example:

ZN003:WALL001:AFN Linkage Node 1 to Node 2 Mass Flow Rate [kg/s](TimeStep)
ZN003:WALL001:AFN Linkage Node 2 to Node 1 Mass Flow Rate [kg/s](TimeStep)

Reply:
@lgu1234 Oops, looks like I got confused about the file I was editing. My local tests had these in there, I'll fix that.